release/nightly_tests/shuffle/shuffle_compute_multi.yaml

```diff
@@ -6,7 +6,7 @@ max_workers: 3
 # TODO(ekl/sang) switch to i2.8xl and mount the NVMe disks
 head_node_type:
     name: head_node
-    instance_type: i3.8xlarge
+    instance_type: i3.4xlarge
```

Comment:
Need to use the same instance for both compute config to override resources in app config.